Redis.cpp

```diff
@@ -715,6 +715,12 @@ RedisSubscribeResult Redis::startSubscribingNonBlocking(RedisMsgCallback message
   return RedisSubscribeSuccess;
 }
 
+bool Redis::ping()
+{
+  TRCMD_EXPECTOK("PING", "OK");
```

Comment:
Additionally argument to `TRCMD_EXPECTOK` are passed through as arguments to the Redis command ("PING"), so in this case it would send "PING OK". If you want to send a bare "PING" without any argument, you'd just want `TRCMD_EXPECTOK("PING");` Is sending with the "OK" argument what you were intending? It's hard to tell without seeing a test for this command...

... which leads me to my next request: please implement a test for this in the [integration tests](https://github.com/electric-sheep-co/arduino-redis/blob/master/test/integration/integration-tests.ino). The pattern is pretty simple and straightforward but please ask if you have trouble.

Reply:
I've read documentation for [PING command](https://redis.io/commands/ping/). Argument in PING command is returned for the successful response.

Ok. I'll implement test... Reply a comment when it will be done.

Reply:
> I've read documentation for [PING command](https://redis.io/commands/ping/). Argument in PING command is returned for the successful response.

Yeah and that's why I was asking, it's completely fine if you wanted to send "OK" and then expect that response in the test, but like I said I wasn't sure without seeing some usage of it. 😊 

Thank you! 🙇